src/transformers/generation_utils.py

```diff
@@ -381,6 +383,12 @@ def generate(
             use_cache: (:obj:`bool`, `optional`, defaults to :obj:`True`):
                 Whether or not the model should use the past last key/values attentions (if applicable to the model) to
                 speed up decoding.
+            beam_groups (:obj:`int`, `optional`, defaults to 1):
+                Number of groups to divide :obj:`num_beams` into in order to ensure diversity among different groups of
+                beams. See `this paper <https://arxiv.org/pdf/1610.02424.pdf>`__ for more details.
```

Comment:
Could we add one sentence like `"To enable diverse beam search, :obj:`beam_groups` should be set to a value larger than 1.`